### ISSUE TITLE
import error using angular/ts

### ISSUE THREAD

**Issue by pablomaurer** [mesqueeb/nestify-anything#16]
When importing the lib in angular I had to remove those lines from package.json:

```
  "exports": {
    ".": "./dist/index.js"
  },
```
else i get following error message:
`Module not found: Error: Package path ./dist is not exported from package /home/user/IdeaProjectsr/app-client/node_modules/nestify-anything (see exports field in /home/user/IdeaProjects/app-client/node_modules/nestify-anything/package.json)`

Imports also shows some different style compared to the flatten-anything package:
```
import { flatten } from 'flatten-anything';
import { nestifyObject } from 'nestify-anything/dist';
```

**Comment by mesqueeb** [comment]
That shouldn't be right. Look at the dist folder, it's set up correct in all my use cases and to my knowledge. 🤔

You need to import from 'nestify-anything' not from 'nestify-anything/dist'

**Comment by pablomaurer** [comment]
Mhm, webstorm did those imports autmaticly. For `flatten` correct but for `nestify` wrong. So there must be a difference in how those two packages are published.

But I don't care anymore as I only stay with the `flatten` package and removed `nestify`, so feel free to close this issue =)

**Comment by mesqueeb** [comment]
This might have been an issue only on 3.0.0 and was already fixed on 3.0.1.
